plugins/BEdita/API/src/Error/ExceptionRenderer.php

```diff
@@ -43,9 +43,10 @@ public function render()
     {
         $isDebug = Configure::read('debug');
 
-        $code = $this->_code($this->error);
-        $message = $this->_message($this->error, $code);
+        $status = $this->_code($this->error);
```

Comment:
A couple of minor observations about `$status` and `$code`:

1) why renaming `$code` to `$status`? it could be `http code` (=> `$httpCode`)
2) Regarding following statement: `$code = $this->appErrorCode($this->error);`:
function `appErrorCode` returns "string Error code". Then instead of `$code`, it could be `$errorCode` or `$appErrorCode`.

Disambiguisation of variables names could help in following `$this->jsonError($status, $title, $detail, $code, $trace)`
[=> `$this->jsonError($httpCode, $title, $detail, $appErrorCode, $trace)`]